Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -272,6 +272,10 @@
 #endif
     };
 
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+    m_acceleratedTimelineTimeOrigin = layerTreeTransaction.acceleratedTimelineTimeOrigin();
```

Comment:
I've realised that this won't work with site-isolation enabled :(

LayerTreeTransactions can be received for both the main frame, as well as any of the isolated frames within it. If both the main frame and an isolated iframe both run accelerated animations, they'll be overwriting each other's value for the timeline origin.

Maybe that's ok for now, but probably worth a FIXME at least, and a radar.